### DISCUSSION TITLE
Searching for locked issues

### DISCUSSION THREAD

**Discussion by jeremystretch** [community/community#23662] [New to GitHub]
<p>According to the documentation <a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-based-on-whether-a-conversation-is-locked" rel="noopener nofollow ugc">here</a>, it should be possible to filter for locked issues by searching for <code>is:issue is:locked</code> within a repo. However, this doesn’t seem to have any effect: All issues are returned. Same for <code>is:unlocked</code>. Am I missing something obvious?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I think this is a lack of clarity in documentation - and we have an issue open to fix that.</p>
<p>The repository level searches are a filter search, and don’t support all search qualifiers. <code>locked</code> is one of the unsupported ones.</p>
<p>If you use the same search in the global search, you should find that it works for you as described.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/8/b851676abcf78c26ba18532b7ac2b39b6f0a25b7.png" data-download-href="/uploads/short-url/qiygeeW7v6TYDYq4f2F5gk0UAbZ.png?dl=1" title="Screenshot 2020-11-03 at 15.53.32"><img src="https://user-images.githubusercontent.com/11161530/181078669-727d1a03-3821-4cc2-a6ca-7299b684367c.png" alt="Screenshot 2020-11-03 at 15.53.32" data-base62-sha1="qiygeeW7v6TYDYq4f2F5gk0UAbZ" width="517" height="31" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b851676abcf78c26ba18532b7ac2b39b6f0a25b7_2_517x31.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b851676abcf78c26ba18532b7ac2b39b6f0a25b7_2_775x46.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/8/b851676abcf78c26ba18532b7ac2b39b6f0a25b7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b851676abcf78c26ba18532b7ac2b39b6f0a25b7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-11-03 at 15.53.32</span><span class="informations">971×60 7.05 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Ex:</p>
<p><code>repo:github/gh-ost is:issue is:unlocked</code></p>
<p>and</p>
<p><code>repo:github/gh-ost is:issue is:locked</code></p>
<p>Should return different sets.</p>

**Comment by jeremystretch** [comment]
<p>Thanks for the quick reply! I didn’t realize there was a distinction between global and repo-level search with regard to supported filters. Good to know!</p>
